docs/installation.rst

```diff
@@ -57,22 +61,10 @@ This will create a ``dist`` directory in ``treeherder-ui`` where concatenated an
 Configuration
 =============
```

Comment:
I'm wondering if this Configuration section might be helpful following the Running the Web Server part above, or if not, perhaps stay here but still provide some context on the sample.local.conf.js which was the origin of the file. Without that, looking at this section it refers to a file, local.conf.js that doesn't exist by default.